contributing/patches/load_local_synthetic_data.patch

```diff
@@ -252,13 +252,13 @@ index a1401b55..a44fca81 100644
        StaticRifResource.SYNTHETIC_BENEFICIARY_1999,
        StaticRifResource.SYNTHETIC_BENEFICIARY_2000,
 diff --git a/apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java b/apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java
-index a163268d..b067192b 100644
+index a163268d..5378d8ce 100644
 --- a/apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java
 +++ b/apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java
-@@ -315,6 +315,19 @@ public final class RifLoaderIT {
-     } finally {
+@@ -316,6 +316,20 @@ public final class RifLoaderIT {
        if (entityManager != null) entityManager.close();
      }
+   }
```

Comment:
Here's the missing curly brace that was causing the build error

Reply:
Awesome. Thank you.